app/controllers/users/sessions_controller.rb

```diff
@@ -9,21 +11,35 @@ def new
   # POST /resource/sign_in
   def create
     super
-    if user_signed_in?
-      UserAction.save_user_action(current_user.id,  UserAction::ACTION_SIGN_IN, new_user_session_path)
-    end
   end
 
   # DELETE /resource/sign_out
   def destroy
-    UserAction.save_user_action(current_user.id,  UserAction::ACTION_SIGN_OUT, UserAction::PATH_SIGN_OUT)
     super
   end
 
-  # protected
+  private
+    def user_log_in
+      if current_user.present?
+        UserAction.save_user_action(current_user.id,  UserAction::ACTION_SIGN_IN, new_user_session_path)
+        cookies[:login_attempts] = 0
+      end
+    end
```

Comment:
Do not wrap method body with `if` if `else` not present.
```ruby
def user_log_in
 return unless current_user
 UserAction.save_user_action(current_user.id,  UserAction::ACTION_SIGN_IN, new_user_session_path)
 cookies[:login_attempts] = 0
end
```

Reply:
Fixed